src/ypd/templates/project.html

```diff
@@ -30,7 +30,7 @@ <h1>{{project.title}}</h1>
         </a>
     </div>
 
-    {% if project.can_be_modified_by(user) %}
+    {% if project.can_be_modified_by(user) and project.archived == False %}
```

Comment:
Why are we protecting archived projects from modification?

Reply:
The almighty gods have spoken